NosTests/Service/NostrIdentifier/NostrIdentifierTests.swift

```diff
@@ -190,7 +190,7 @@ class NostrIdentifierTests: XCTestCase {
     }
 
     /// Example taken from note1tyt3a7tqxhgvvve87jzxpsndguw6vq29qfuenau40kartujlrppq5zmncg
-    func test_naddr_4() throws {
+    func test_naddr_with_no_relays() throws {
```

Comment:
One more test here? Can we check that an naddr that is missing a kind fails to parse?

Reply:
Yes indeed. But how can I create a `naddr` that's otherwise valid but missing the `kind`? I know `nak` doesn't support that. I can just make a 100% invalid one -- I think that's about the best we can do here. I'll do that.

Reply:
Added a test for invalid `naddr`.